<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>СКТ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonOpenDialogAreaAndAnomaly">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>520</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Ввод данных
задачи</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSolve">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>590</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Решить</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBlocks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>821</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxAlpha">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>α Регуляризация</string>
    </property>
    <widget class="QLabel" name="labelAlpha">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>α = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAlpha">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxGamma">
    <property name="geometry">
     <rect>
      <x>849</x>
      <y>110</y>
      <width>181</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>γ Регуляризация</string>
    </property>
    <widget class="QCheckBox" name="checkBoxUseGamma">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Использовать γ</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelGamma0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>γ0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditGamma0">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1e-15</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelGammaD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>dγ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditGammaD">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditGammaPDiff">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelGammaPDiff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Допустимое
отличие
намагничнности</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditGammaFiDiff">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelGammaFiDiff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Максимальное
увеличение
функционала</string>
     </property>
    </widget>
   </widget>
   <widget class="QChartView" name="widgetChart" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>821</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
